woocommerce/woocommerce-admin-test-helper
# Add support for email notes and different note layouts

The changes added in this PR allow the creation of notes type: `info`, `update`, and `email`.
Also it adds support for the creation of notes with layout: `plain`, `banner`, and `thumbnail`.

### Screenshots

![screenshot-one wordpress test-2021 04 23-16_38_51](https://user-images.githubusercontent.com/1314156/115921864-79a40d00-a452-11eb-9cad-6dc4db8b5390.png)

### Detailed test instructions:

- Go to `Tools` > `WCA Test Helper` > `Admin notes` (`/wp-admin/tools.php?page=woocommerce-admin-test-helper`).
- Choose the kind of note you want and press `Add admin note`.
- The note should have been created. To verify this, look for your note name in the DB table `wp_wc_admin_notes`.
